spec/src/main/asciidoc/fallback.asciidoc

```diff
@@ -94,5 +94,28 @@ When `fallbackMethod` is used a `FaultToleranceDefinitionException` will be thro
 The parameter `value` and `fallbackMethod` on `@Fallback` cannot be specified at the same time.
 Otherwise, the `FaultToleranceDefinitionException` exception will be thrown.
 
-The fallback should be triggered when an exception occurs.
-For instance, `BulkheadException`, `CircuitBreakerOpenException`, `TimeoutException` should trigger the fallback.
+The fallback might be triggered when an exception occurs (e.g. `BulkheadException`, `CircuitBreakerOpenException`, `TimeoutException`, etc), detailed below.
```

Comment:
This hasn't changed in this pull request, but I don't like that all the examples here are Fault Tolerance exceptions, it might give the impression that fallback applies only to fault tolerance exceptions when it applies equally to any exception.

Given that you've now added more detail about how fallback works when an exception is thrown, maybe we could take these examples out. We could also add a sentence in an appropriate place to say that exceptions thrown by other fault tolerance policies can be handled by a fallback on the same method as that's what I think we were trying to hint at with these examples.

Reply:
I've rephrased the sentence.